google-client-loader.html

```diff
@@ -162,19 +162,22 @@
 
       _fireSuccess: function() {
         this.fire(this.successEventName,
-            { 'name': this.name, 'version': this.version });
+            { 'name': this.name, 'version': this.version },
+            { composed: true });
```

Comment:
I think you need to manually create and dispatch this event and the ones below if they should be composed because [Polymer 1's `fire`](https://github.com/Polymer/polymer/blob/8ce02609a9ead11b72203bb8036f9d9cee2e1e13/src/standard/utils.html#L219) doesn't actually give the `options` argument to the `Event` constructor. Is this `composed: true` meant for Polymer 1 + Shadow DOM v1?